controllers/gameController.js

```diff
@@ -11,80 +11,91 @@ const dateFormat = "unix"; // iso | unix
+        let gameIds = (await db.checkGames()).map((game) => game.game_id);
 
-    const gamesToPost = upcomingGames.filter((game) => {
-      return !gameIds.includes(game.id);
-    });
+        const gamesToPost = upcomingGames.filter((game) => {
+            return !gameIds.includes(game.id);
+        });
 
-    for (let i = 0; i < gamesToPost.length; i++) {
-      let { outcomes } = gamesToPost[i].bookmakers[0].markets[0];
-      let home_moneyline;
-      let away_moneyline;
-      let {
-        id: game_id,
-        home_team,
-        away_team,
-        commence_time: start_time,
-      } = gamesToPost[i];
+        for (let i = 0; i < gamesToPost.length; i++) {
+            let {outcomes} = gamesToPost[i].bookmakers[0].markets[0];
+            let home_moneyline;
+            let away_moneyline;
+            let {
+                id: game_id,
+                home_team,
+                away_team,
+                commence_time: start_time,
+            } = gamesToPost[i];
 
-      const determineHomeMoneyline = () => {
-        for (let i = 0; i < 2; i++) {
-          if (outcomes[i].name === home_team) {
-            home_moneyline = outcomes[i].price;
-            away_moneyline = outcomes[Math.abs(i - 1)].price;
-          } else {
-            home_moneyline = outcomes[Math.abs(i - 1)].price;
-            away_moneyline = outcomes[i].price;
-          }
+            const determineHomeMoneyline = () => {
+                for (let i = 0; i < 2; i++) {
+                    if (outcomes[i].name === home_team) {
+                        home_moneyline = outcomes[i].price;
+                        away_moneyline = outcomes[Math.abs(i - 1)].price;
+                    } else {
+                        home_moneyline = outcomes[Math.abs(i - 1)].price;
+                        away_moneyline = outcomes[i].price;
+                    }
+                }
+            };
+            determineHomeMoneyline();
+            await db.addGames(
+                game_id,
+                home_team,
+                away_team,
+                home_moneyline,
+                away_moneyline,
+                start_time
+            );
         }
-      };
-      determineHomeMoneyline();
-      await db.addGames(
-        game_id,
-        home_team,
-        away_team,
-        home_moneyline,
-        away_moneyline,
-        start_time
-      );
+        res.status(200).json({message: `Added games to db`});
+    } catch (error) {
+        console.log(error);
+        res.status(404).json({message: "Error posting games"});
     }
-    res.status(200).json({ message: `Added games to db` });
-  } catch (error) {
-    console.log(error);
-    res.status(404).json({ message: "Error posting games" });
-  }
+});
+
+router.get("/upcoming", async (req, res) => {
+    // const games = await db.getUpcomingGames();
+    // console.log("Game", games);
+    // res.json(games);
+    const {rows} = await db.query("SELECT * FROM games");
```

Comment:
To get games that are actually upcoming and not just all games, there could be  something like "where game-start_time < current_time" so every_time we query the db to render the frontend, we only send games that haven't started. That way we can render possible bets correctly"